README.md

```diff
@@ -1,129 +1,75 @@
-# Azure Functions Plugin 
+# Azure Functions Serverless Plugin 
 
 This plugin enables Azure Functions support within the Serverless Framework.
 
 ## Getting started
 
-### 1. Get a Serverless Service with Azure as the Provider
+### Pre-requisites
 
-1. Recommend using Node v6.5.0
-1. Install the serverless tooling - `npm i -g serverless`
-1. Create boilerplate (change `my-app` to whatever you'd prefer): `serverless install --url https://github.com/azure/boilerplate-azurefunctions --name my-app`
-1. `cd my-app`
-2. `npm install`
+1. Node.js v6.5.0+ *(this is the runtime version supported by Azure Functions)*
+2. Serverless CLI. You can run `npm i -g serverless` if you don't already have it.
+3. An Azure account. If you don't already have one, you can sign up for a [free trial](https://azure.microsoft.com/en-us/free/) that includes $200 of free credit.
 
-### 2. Set up credentials
+### Create a new Azure service
 
-We'll set up an Azure Subscription and our service principal. You can learn more in the [credentials doc]( https://www.serverless.com/framework/docs/providers/azure/guide/credentials).
+1. Create a new service using the standard Node.js template: `serverless create -t azure-nodejs --name <APP_NAME>`
+2. CD into the generated app directory: `cd <APP_NAME>`
```

Comment:
Actually, I just need to change this sample to use `-p` instead, and it will create the directory as expected. This was a doc bug.